mifosng-android/src/main/java/com/mifos/mifosxdroid/online/documentlist/DocumentListPresenter.java

```diff
@@ -114,11 +116,22 @@ public void onCompleted() {
                     @Override
                     public void onError(Throwable e) {
                         getMvpView().showProgressbar(false);
-                        getMvpView().showFetchingError(R.string.failed_to_remove_document);
+                        if (e instanceof HttpException) {
```

Comment:
why don't you use this https://github.com/openMF/android-client/blob/master/mifosng-android/src/main/java/com/mifos/utils/MFErrorParser.java#L25